src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/Client/StorageClient.cs

```diff
@@ -141,12 +143,93 @@ public async Task<Instance> GetInstances(string instanceId, int instanceOwnerId)
         public async Task<string> PostInstances(string applicationId, int instanceOwnerId)
         {
             string requestUri = $"{versionPrefix}/instances?applicationId={applicationId}&instanceOwnerId={instanceOwnerId}";
-            
+
             HttpResponseMessage createInstanceResponse = await client.PostAsync(hostName + requestUri, string.Empty.AsJson());
             string newId = await createInstanceResponse.Content.ReadAsStringAsync();
 
             return newId;
         }
+
+        /// <summary>
+        /// Retrieves all instance events related to given instance id from instanceEvent collection.
+        /// </summary>
+        /// <param name="instanceId"> Id of instance to retrieve events for. </param>
+        /// <returns>List of instance events.</returns>
+        public async Task<List<InstanceEvent>> GetAllInstanceEvents(string instanceId)
+        {
+            string requestUri = $"{versionPrefix_new}/instanceEvents/?instanceId={instanceId}";
```

Comment:
We agreed on technical forum to keep all paths in urls lower-case. Parameters should be lowerUppercase. Two things:
1) Separate class for EventClient
2) path /instance-events? (no / before ?)

Reply:
Paths updated. All lower case